srcpkgs/gnustep-base/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'gnustep-base'
 pkgname=gnustep-base
-version=1.25.1
-revision=3
+version=1.26.0
+revision=1
 build_style=gnu-configure
-hostmakedepends="gnustep-make gcc-objc"
+hostmakedepends="gnustep-make gcc-objc pkg-config"
 makedepends="libobjc-devel libffi-devel libxml2-devel gnutls-devel
  libxslt-devel icu-devel"
 short_desc="Library of general-purpose Objective C objects for GNUstep"
 maintainer="Andrew Benson <abenson+void@gmail.com>"
 license="GPL-3"
```

Comment:
please use SPDX short-identifier

Reply:
Good catch, when back to check and there are parts that are 2+ and others that 3+